assembly/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/WsMasterModule.java

```diff
@@ -110,6 +110,14 @@
 
   @Override
   protected void configure() {
+    // Workaround for https://github.com/fabric8io/kubernetes-client/issues/2212
+    // OkHttp wrongly detects JDK8u251 and higher as JDK9 which enables Http2 unsupported for JDK8.
+    // Can be removed after upgrade to Fabric8 4.10.2 or higher or to Java 11
+    if (System.getProperty("java.version", "").startsWith("1.8")
+        && System.getenv("HTTP2_DISABLE") == null) {
+      System.setProperty("http2.disable", "true");
```

Comment:
this property will adjust this configuration https://github.com/fabric8io/kubernetes-client/blob/f94496b9e117b09e841db5cfecd94e4accf2ce00/kubernetes-client/src/main/java/io/fabric8/kubernetes/client/Config.java#L374